tasks/xgettext.js

```diff
@@ -27,13 +27,15 @@ module.exports = function(grunt) {
 
```

Comment:
Consider switch to let/const over var, especially where inline vars that are really hoisted can lead to initialization confusion.